ocw/lib/db.py

```diff
@@ -222,6 +229,51 @@ def update_run():
         init_cron()
 
 
+def is_delete_instance_allowed(instance):
+    return 'openqa_created_by' in instance.csp_info
+
+
+def delete_instance(instance):
+    if not is_delete_instance_allowed(instance):
+        raise PermissionError('This instance isn\'t managed by openqa')
```

Comment:
not sure that we really want to raise exception here , maybe skipping is just enough ?
But in case we do let's put csp_info , because we use it to decide to raise this exception

Reply:
I keep the exception but added csp_info